flink-table/flink-sql-jdbc-driver/pom.xml

```diff
@@ -84,6 +84,12 @@
 			<version>${project.version}</version>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>com.google.guava</groupId>
+			<artifactId>guava</artifactId>
+			<version>${guava.version}</version>
+			<scope>provided</scope>
+		</dependency>
```

Comment:
Suddenly with Calcite 1.31.0 it starts failing with guava's `ImmutableList` not found.

Reply:
yes, sort of.
After closer look I realized that before 1.31.0 `FlinkSqlParserImpl` didn't depend on guava. And with 1.31.0 after these 2 commits
https://github.com/apache/calcite/commit/7c1e2746b94a50f0bb08571287acb5327fdc1b16
https://github.com/apache/calcite/commit/75d064f659a038371d84f9fdba7847e1631d1f1d
it starts depending on guava and as a result starts requiring same dependency as for tests

Reply:
But why is `flink-sql-jdbc-driver` is affected here? We should change the pom.xml of the `flink-sql-parser`? It seems `flink-sql-jdbc-driver` is just a symptom, not the root cause.

Reply:
looks like 100% will not work since sql client explicitely depends on commons-io, so this one is currently skipped,
all others are applied.
May be it makes sense to remove commons-io dep (only one method) within a follow up task